File: src/main/java/RemoveElement/GaryLinTouchWorld/RemoveElement.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    var length = nums.length;
    for(let i = 0; i<length ; i++){
        if(nums[i] === val){
            nums.splice(i,1);
            i -=1;
            length -=1;
        }
    }
    return length;
};

File: src/main/java/RemoveElement/AzusaMutan/RemoveElement.java
public class RemoveElement {
    public int removeElement(int[] nums, int val) {
        int res = 0;
        for (int n: nums
        ){
            if ((val ^ n) != 0) {
                nums[res++] = n;
            }
        }

        return res;
    }
}
